### PR TITLE
feat: add more stop reason handle (#51)

### PR DESCRIPTION
**Pull request by tkhmy**
* feat: add more stop reason handle

Signed-off-by: khtan <tkh.my.p@gmail.com>

* change voice

Signed-off-by: khtan <tkh.my.p@gmail.com>

Signed-off-by: khtan <tkh.my.p@gmail.com>

## PR Type

- Improvement

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

**Review by yabuta — APPROVED** [review]

File: src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py [hunk 1]
```diff
@@ -270,10 +270,14 @@ def sub_stop_reason(self, stop_reason):
                 self.send_announce("obstacle_stop")
                 self._stop_reason_announce_time = self._node.get_clock().now()
             elif shortest_stop_reason in [
-                "StopLine",
-                "Walkway",
-                "Crosswalk",
+                "Intersection",
```
Comment (h-ohta) [inline src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py:273]: （日本語で失礼します）
このように指定する形にするのであれば、パラメータ化しておいたほうが柔軟性がありそうです

Comment (tkhmy) [reply src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py:273]: なるほど、修正します。

Comment (tkhmy) [reply src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py]: @h-ohta [d0ad09d](https://github.com/tier4/vehicle_voice_alert_system/pull/52/commits/d0ad09d316697e24bf4870541d1f668967e6b117)で修正しました。

**Review by h-ohta — APPROVED** [review]
LGTM

**Comment by h-ohta** [comment]
TIERIV INTERNAL LINK: https://tier4.atlassian.net/browse/T4PB-23652

**Pull request merged**